src/middlewares/login.Vendedores.js

```diff
@@ -0,0 +1,30 @@
+const bcrypt = require("bcryptjs");
+
+const LoginVendedor = async (req, res) => {
+  const { clave, password } = req.body;
+
+  try {
+    const Vendedor = await Vendedor.findOne({ clave });
+    if (!Vendedor) {
+      return res.status(400).json({ message: "vendedor no encontrado" });
```

Comment:
Se utiliza el status 404 cuando algo no se encuentra. 
```suggestion
      return res.status(404).json({ message: "vendedor no encontrado" });
```